### PR TITLE
Chore: Minor polishing

### PR DESCRIPTION
**Pull request by facuspagnuolo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+![Aragon Protocol](./docs/aragon-protocol.png)
+
+<img align="right" src="https://github.com/aragon/protocol/workflows/CI/badge.svg">
+  <a href="https://github.com/aragon/protocol/actions"/>
+</img>
+
+## Project
+
+#### 👩‍️ [Become an Aragon Protocol guardian](https://ant.aragon.org)
```
Comment (sohkai) [inline README.md:9]:
```suggestion
#### 👩‍️ [Become an Aragon Protocol guardian](https://aragon.org/token/ant)
```

URL might change in the future, but I'd use that one for now :shrug:

File: README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+Aragon Protocol is now live on Ethereum mainnet. You can become a guardian by staking ANT.
```
Comment (sohkai) [inline README.md:10]:
```suggestion
Aragon Protocol will soon be live on Ethereum mainnet. You can become a guardian by staking ANT.
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+#### ⚖ [Check out the Aragon Protocol Dashboard](https://protocol.aragon.org)
+The Aragon Protocol Dashboard is the central app where all dispute-related information is available for guardians.
+
+#### 📚 [Read the User Guide](https://help.aragon.org/category/47-aragoncourt)
```
Comment (sohkai) [inline README.md:15]: Can probably leave this out for now and add an issue to re-add it later?

File: README.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+#### ⚖ [Check out the Aragon Protocol Dashboard](https://protocol.aragon.org)
+The Aragon Protocol Dashboard is the central app where all dispute-related information is available for guardians.
+
+#### 📚 [Read the User Guide](https://help.aragon.org/category/47-aragoncourt) 
+Read the user guide if you have any doubts about the Aragon protocol, Protocol Dashboard, or related tools.
+
+## Protocol
+
+#### 📓 [Read the full documentation](/docs)
+Aragon Protocol is a dispute resolution protocol that runs on Ethereum. It's one of the core components of the [Aragon Network](https://aragon.org/network/).
```
Comment (sohkai) [inline README.md:21]:
```suggestion
Aragon Protocol is a dispute resolution protocol running on Ethereum. It is the core component of the [Aragon Network](https://aragon.org/network/).
```

😉

File: README.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+#### 🚧 Project stage: v1 implementation
+After a long research and development phase, Aragon Protocol's v1 implementation has been [released](https://www.npmjs.com/package/@aragon/protocol) and [deployed](https://etherscan.io/address/0xee4650cBe7a2B23701D416f58b41D8B76b617797#code).
```
Comment (sohkai) [inline README.md:24]:
```suggestion
After a long research and development phase, Aragon Protocol's v1 implementation will soon be released as an upgraded version of Aragon Court.
```

File: README.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+#### ✅ Security review status: audited
```
Comment (sohkai) [inline README.md:26]: Not sure about this one; maybe just refer to the Court's previous audit for now?

File: README.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+
+#### 👋 Get started contributing with a [good first issue](https://github.com/aragon/aragon-protocol/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)
+Don't be shy to contribute even the smallest tweak. Everyone will be especially nice and helpful to beginners to help you get started!
+
+## Deployed instances
+
+#### Mainnet
+
+The mainnet instance of Aragon Protocol is deployed at [`0xee4650cBe7a2B23701D416f58b41D8B76b617797`](https://etherscan.io/address/0xee4650cBe7a2B23701D416f58b41D8B76b617797#code)
+
```
Comment (sohkai) [inline README.md:37]: Should we remove this section for now?

File: README.md [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+#### Testing
+
+There are a few testing instances deployed of Aragon Protocol, please refer to the [testing guide](/docs/8-testing-guide) to have a better understanding on using these.
+
+## Help shape Aragon Protocol
+- Discuss in [Aragon Forum](https://forum.aragon.org/tags/dispute-resolution)
+- Join the [Aragon Protocol channel](https://discord.gg/nxMejdG) on Discord.
```
Comment (sohkai) [inline README.md:44]:
```suggestion
- Join the [Aragon Discord server](https://discord.com/invite/aragon).
```

File: docs/1-mechanism/readme.md [hunk 9]
```diff
@@ -16,11 +16,11 @@ Even though the Aragon Protocol could theoretically resolve any type of binary d
 
 ### 1.2. High-level flow
 
-- Guardians deposit ANT into a bonding curve to generate ANJ tokens.
-- Guardians stake ANJ to the Protocol contract and schedule their activation and deactivation for the time period in which they can be drafted to rule on disputes.
+- Guardians deposit ANT into a bonding curve to generate ANT tokens.
```
Comment (sohkai) [inline docs/1-mechanism/readme.md:19]: I think we can safely remove this for now :)

File: packages/evm/contracts/lib/os/ERC20.sol [hunk 10]
```diff
@@ -1,7 +1,7 @@
 // Brought from https://github.com/aragon/aragonOS/blob/v4.3.0/contracts/lib/token/ERC20.sol
```
Comment (sohkai) [inline packages/evm/contracts/lib/os/ERC20.sol:1]: Given that we're more or less jettisoning aragonOS, how do you feel about changing these to be just in `lib` or even modified to use their OZ-equivalent?

File: packages/evm/contracts/AragonProtocol.sol [hunk 11]
```diff
@@ -115,9 +115,10 @@ contract AragonProtocol is Controller, IArbitrator {
     * @return subject Arbitrable instance associated to the dispute
     * @return ruling Ruling number computed for the given dispute
     */
-    function rule(uint256 _disputeId) external returns (IArbitrable subject, uint256 ruling) {
+    function rule(uint256 _disputeId) external returns (address subject, uint256 ruling) {
         IDisputeManager disputeManager = IDisputeManager(_getDisputeManager());
-        (subject, ruling) = disputeManager.computeRuling(_disputeId);
+        (IArbitrable _subject, uint8 _ruling) = disputeManager.computeRuling(_disputeId);
```
Comment (sohkai) [inline packages/evm/contracts/AragonProtocol.sol:120]: Did we update the ruling options to be only uint8? TBH I think anything more than 2/4 options is going to need explicit documentation, so at that point, we might as well just use uint256 to save some gas 😄

**Review by sohkai — APPROVED** [review]
A few more minor comments / adjustments for now, otherwise all good for me ❤️!

File: README.md (hunk 8 above)
Comment (facuspagnuolo) [reply README.md:44]: Addressed in 6bbd771b6f5d7c3da49fa891bb119e3c16649a20

File: README.md (hunk 7 above)
Comment (facuspagnuolo) [reply README.md:37]: Addressed in 6bbd771b6f5d7c3da49fa891bb119e3c16649a20

File: README.md (hunk 6 above)
Comment (facuspagnuolo) [reply README.md:26]: Addressed in 6bbd771b6f5d7c3da49fa891bb119e3c16649a20

File: README.md (hunk 5 above)
Comment (facuspagnuolo) [reply README.md:24]: Addressed in 6bbd771b6f5d7c3da49fa891bb119e3c16649a20

File: README.md (hunk 4 above)
Comment (facuspagnuolo) [reply README.md:21]: Addressed in 6bbd771b6f5d7c3da49fa891bb119e3c16649a20

File: README.md (hunk 3 above)
Comment (facuspagnuolo) [reply README.md:15]: Addressed in 6bbd771b6f5d7c3da49fa891bb119e3c16649a20

File: README.md (hunk 2 above)
Comment (facuspagnuolo) [reply README.md:10]: Addressed in 6bbd771b6f5d7c3da49fa891bb119e3c16649a20

File: README.md (hunk 1 above)
Comment (facuspagnuolo) [reply README.md:9]: Addressed in 6bbd771b6f5d7c3da49fa891bb119e3c16649a20

File: docs/1-mechanism/readme.md (hunk 9 above)
Comment (facuspagnuolo) [reply docs/1-mechanism/readme.md:19]: Addressed in 81a4818b240c373ba3faa712d08cc94b7d06a181

File: packages/evm/contracts/lib/os/ERC20.sol (hunk 10 above)
Comment (facuspagnuolo) [reply packages/evm/contracts/lib/os/ERC20.sol:1]: I have just read this but I think I did this in #18

File: packages/evm/contracts/AragonProtocol.sol (hunk 11 above)
Comment (facuspagnuolo) [reply packages/evm/contracts/AragonProtocol.sol:120]: I think the main reason is to avoid having one extra slot when creating disputes

**Pull request merged**